src/main/java/uk/gov/hmcts/reform/draftstore/data/DraftStoreDAO.java

```diff
@@ -68,6 +69,16 @@ public void insert(String userId, CreateDraft newDraft) {
         );
     }
 
+    public void update(int id, String userId, UpdateDraft draft) {
+        jdbcTemplate.update(
+            "UPDATE draft_document SET document = cast(:doc AS JSON), document_type = :type WHERE id = :id",
```

Comment:
doc::json is simpler any reason you can't do that?

Reply:
👍 fixed